src/SplitButton/index.scss

```diff
@@ -0,0 +1,90 @@
+@import "../Button/variants.scss";
+@import "../Button/sizes.scss";
+@import "../Button/states.scss";
+
+.nds-splitButton {
+  display: inline-flex;
+  min-height: fit-content;
+  align-items: stretch;
+
+  .nds-button {
+    border-top-right-radius: 0 !important;
+    border-bottom-right-radius: 0 !important;
+  }
+}
+
+.nds-splitButton-action {
```

Comment:
The `nds-splitButton-action` is the element we apply the Sass placeholders to so it matches the `Button` component we compose into the left side of `SplitButton`